base/precise-code-intel/worker.Deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    description: Precise code intelligence services for Sourcegraph
+  labels:
+    deploy: sourcegraph
+  name: pci-worker
+spec:
+  minReadySeconds: 10
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: pci-worker
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: pci-worker
+    spec:
+      containers:
+      - env:
+        - name: PCI_NUM_APIS
+          value: '0'
+        - name: PCI_NUM_BUNDLE_MANAGERS
+          value: '0'
+        - name: PCI_NUM_WORKERS
+          value: '1'
```

Comment:
Can we have the default here be 2?

I would like us to generally use the same scaling mechanisms across kubernetes & docker-compose deployments where possible, and having this be 2 means we won't e.g. accidentally break the in-container replication without noticing. It'll also push us to scale workers using this in k8s deployments (instead of using replicas) which will inherently make us handle monitoring of these containers better.

Reply:
Both APIs and workers are up to 2.